KidtronicsLanguage/var.h

```diff
@@ -43,6 +53,10 @@ class var {
     
     // Assignment operator.
     var& operator=(const var& other);
+    // Get value in double
+    double getValueInDouble() const;
```

Comment:
Make this function private, user of the class will never need this.

Reply:
It will be used in other class, for example, left hand side try to compare itself with right hand side.